src/py/flwr/client/app.py

```diff
@@ -612,6 +612,17 @@ def _on_backoff(retry_state: RetryState) -> None:
                     send(reply_message)
                     log(INFO, "Sent reply")
 
+                except RunNotRunningException:
+                    log(INFO, "")
+                    log(
+                        INFO,
+                        "SuperNode stopped execution. "
```

Comment:
I think it would be better to say "ClientApp stopped execution" or "SuperNode aborted sending the reply message," as the SuperNode won't stop after this exception. This might cause slight confusion.

Reply:
Good point. I've rephrased to "SuperNode aborted sending the reply message.".